lib/admin/components/ServerSettings.js

```diff
@@ -149,6 +150,10 @@ class ServerSettings extends Component<Props, State> {
     }
   }
 
+  _onExpandServer = (index: number) => {
+    this.props.fetchServer(this.state.otpServers[index].id)
+  }
+
   _onSaveServer = (index: number) => this.props.updateServer(this.state.otpServers[index])
```

Comment:
What happens if a server is expanded, you apply updates, and click save? Are the server ec2 instances no longer shown?